### DISCUSSION TITLE
VT designer

### DISCUSSION THREAD

**Discussion by freggala** [Open-Agriculture/AgIsoStack-plus-plus#315] [General]
The seeder example has a neat VT that comes in the BasePool.iop I guess. How was it created? Do you know of any (functional) FOSS VT-Design tools?

**Comment by wynrbwa** [comment]
Hello,Do you mean if there is a vt-designer software for free?Ummm I think many Isobus solution providers sell their service now,so I guess there is not a free and stable software like this:)

**Comment by jeph36** [comment]
@freggala You could take a look at PoolEdit as a FOSS VT pool designer. It has probably been 10 years since I tested it or looked at what it supports. I know it has been used in commercial applications in the past, probably with additional proprietary scripts or add-ons.
https://github.com/moehman/PoolEditDist
